sbi/inference/posteriors/mcmc_posterior.py

```diff
@@ -411,6 +411,15 @@ def sample_batched(
         assert (
             method == "slice_np_vectorized"
         ), "Batched sampling only supported for vectorized samplers!"
+        # warn if num_chains is larger than num requested samples
+        if num_chains > torch.Size(sample_shape).numel():
```

Comment:
I would prefer not to raise a warning because it would be raised with every call. I added a clearer description in the doc string and I think that should suffice.

Reply:
How about raising a warning if `num_chains_extended` is very big?

Reply:
good idea. How big, `num_chains_extended>100`?